benchmark/benchmark-atomic.cpp

```diff
@@ -0,0 +1,220 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+// RAJA/RAJA.hpp cannot be included here because the include logic will
+// default all atomic implementations to a desul backend.
+#include "RAJA/policy/openmp/atomic.hpp"
+#include "RAJA/RAJA.hpp"
+#include "RAJA/policy/openmp/policy.hpp"
+
+#include <type_traits>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+
+/// Conditional compilation for CUDA benchmarks.
+#if defined RAJA_ENABLE_CUDA
```

Comment:
```suggestion
#if defined(RAJA_ENABLE_CUDA)
```

Although parens are not required, please use them for consistency with other RAJA code.